### PR TITLE
Check for invalid private keys in wallet_decrypt_unsafe

### PR DESCRIPTION
**Pull request by SergiySW**
I.e. corrupted database

### CODE REVIEW

**Comment by PlasmaPower** [comment]
We already check for this in `rai::wallet_store::fetch`, right?

**Comment by SergiySW** [comment]
Yes, and return general bool error that can mean several different things. Also not showed to user, only assert

**Comment by PlasmaPower** [comment]
Ok, then this change seems good. :+1:

File: rai/node/cli.cpp [hunk 1]
```diff
@@ -433,7 +433,14 @@ std::error_code rai::handle_node_options (boost::program_options::variables_map
...
 							auto error (existing->second->store.fetch (transaction, account, key));
 							assert (!error);
-							std::cout << boost::str (boost::format ("Pub: %1% Prv: %2%\n") % account.to_account () % key.data.to_string ());
+							if (rai::pub_key (key.data) == account)
+							{
+								std::cout << boost::str (boost::format ("Pub: %1% Prv: %2%\n") % account.to_account () % key.data.to_string ());
+							}
+							else
+							{
+								std::cout << boost::str (boost::format ("Pub: %1% Prv: %2% Invalid private key!\n") % account.to_account () % key.data.to_string ());
```
Comment (rkeene) [inline rai/node/cli.cpp]: Should this be stderr, rather than stdout ?

Comment (SergiySW) [reply rai/node/cli.cpp]: If add "Invalid private key" to different line

**Pull request merged**